_data/projects/OskiBot.yml

```
@@ -0,0 +1,10 @@
+name: OskiBot
+desc: An all-purpose chatbot for UC Berkeley students built on Cisco's Webex Teams messaging application
+site: https://github.com/mbzhu1/oskibot
+tags:
+  - up-for-grabs
+  - hacktoberfest
+  - enhancement
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/mbzhu1/oskibot/issues/2
```

This link should be to the issues labelled with `up-for-grabs`:

```suggestion
  link: https://github.com/mbzhu1/oskibot/labels/up-for-grabs
```